gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/spring/CommonConfiguration.java

```diff
@@ -223,7 +224,8 @@ public EmailService emailService() {
                 mfaChallengeExpireAfter,
                 mfaVerifyAttemptSubject,
                 registrationVerifySubject,
-                Math.toIntExact(userRegistrationVerifyTimeUnit.toSeconds(userRegistrationVerifyTimeValue))
+                Math.toIntExact(userRegistrationVerifyTimeUnit.toSeconds(userRegistrationVerifyTimeValue)),
+                new DomainServiceImpl("http://localhost:8092")
```

Comment:
**issue**: I don't think it is a good think to create a new instance of the DomainService here as this service is a Spring Component (instantiate as SIngleton), it maybe better to provide tthe service (maybe with Lazy ?) but by doing so, should we introduce cyclic dependency into the bean injection :thinking: